ssvsigner/cmd/ssv-signer/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"log"
+
+	"github.com/alecthomas/kong"
+	"github.com/ssvlabs/ssv/operator/keys"
+	"github.com/ssvlabs/ssv/operator/keystore"
+	"github.com/valyala/fasthttp"
+	"go.uber.org/zap"
+
+	"github.com/ssvlabs/ssv/ssvsigner/server"
+	"github.com/ssvlabs/ssv/ssvsigner/web3signer"
+)
+
+type CLI struct {
+	ListenAddr              string `env:"LISTEN_ADDR" default:":8080"` // TODO: finalize port
+	Web3SignerEndpoint      string `env:"WEB3SIGNER_ENDPOINT"`
```

Comment:
The `WEB3SIGNER_ENDPOINT` should be marked as required to prevent missing values.

Suggested fix:
```go
Web3SignerEndpoint string `env:"WEB3SIGNER_ENDPOINT,required"`
```

Additionally, consider validating the format to ensure it's a valid URL:
```go
if _, err := url.ParseRequestURI(cli.Web3SignerEndpoint); err != nil {
    logger.Fatal("invalid WEB3SIGNER_ENDPOINT format", zap.Error(err))
}
```

Reply:
`kong` doesn't have the `,required` modifier, did you mean `env:"WEB3SIGNER_ENDPOINT" required:""`?

Reply:
added URL validation